zipline-loader/src/test/kotlin/app/cash/zipline/loader/TopologicalSortTest.kt

```diff
@@ -0,0 +1,46 @@
+package app.cash.zipline.loader
+
+import com.google.common.truth.Truth.assertThat
+import kotlin.test.assertFailsWith
+import org.junit.Test
+
+class TopologicalSortTest {
+  @Test
+  fun happyPath() {
+    assertThat(listOf("a", "b", "c")
+      .topologicalSort(edges()))
+      .containsExactly("a", "b", "c")
+      .inOrder()
+    assertThat(listOf("a", "b")
+      .topologicalSort(edges("ba")))
```

Comment:
Do I read this as b depends on a?
I think I should read it that way, because that’s how we specify it in the manifest?

But then I think the edges function is backwards?